frontend/src/store/modules/configForm.js

```diff
@@ -207,7 +209,7 @@ export default {
       } catch (error) {
         context.commit(
           'setConfigSubmissionError',
-          context.state.usingWebadeConfig ? 'An error occurred while attempting to update the application configuration in WebADE.'
+          context.state.commonServiceType === 'webade' ? 'An error occurred while attempting to update the application configuration in WebADE.'
```

Comment:
May be good to have this be set to a constant value as well instead of hardcoded

Reply:
Yeah indeed. There is a constant I used elsewhere, missed this spot.
Fixed